webpack/assets/javascripts/react_app/components/Layout/components/TaxonomyDropdown.js

```diff
@@ -47,7 +47,7 @@ const TaxonomyDropdown = ({
         <li key={i}>
           <a
             className={`${taxonomyType.toLowerCase()}_menuitem`}
-            id={`aid_taxonomy_${taxonomy.title}`}
+            id={`select_taxonomy_${taxonomy.title}`}
```

Comment:
changed the id for the taxonomy selector to make sure we can clean up all that aid references everywhere.